packages/serverpod/test/database/columns/column_enum_test.dart

```diff
@@ -85,4 +89,43 @@ void main() {
       });
     });
   });
+
+  group('Given a ColumnEnum serialized as name', () {
```

Comment:
Suggestion: Expand this to include all methods for the enum.

To ensure that it is compatible.